flink-connectors/flink-connector-aws-kinesis-data-streams/src/test/java/org/apache/flink/connector/kinesis/sink/KinesisDataStreamsSinkITCase.java

```diff
@@ -310,7 +533,15 @@ public Scenario withPartitionKeyGenerator(
             return this;
         }
 
+        public Scenario withProperties(Properties properties) {
+            this.properties = properties;
+            return this;
+        }
+
         private void prepareStream(String streamName) throws Exception {
+            if (streamName.equals("do-not-create-new-stream")) {
+                return;
+            }
```

Comment:
nit: This seems like a hack? Can we update the Scenario to disable stream creation instead? You could skip if stream name is null for example? At least you could extract this String to a constant since it is heavily reused